include/oneapi/dpl/pstl/hetero/dpcpp/sycl_defs.h

```diff
@@ -55,13 +56,41 @@ using __no_init =
     sycl::property::noinit;
 #endif
 
-template <typename _T>
-using __plus =
+#if _ONEDPL_SYCL2020_KNOWN_IDENTITY_PRESENT
+template <typename _BinaryOp, typename _T>
+using __known_identity = sycl::known_identity<_BinaryOp, _T>;
+
+template <typename _BinaryOp, typename _T>
+using __has_known_identity = sycl::has_known_identity<_BinaryOp, _T>;
```

Comment:
I think this definition of `__has_known_identity` is rather useless if in practice we have to further narrow it down to only certain types in `unseq_backend_sycl.h`, and there is no other uses of this trait. Why not just define it here in a way that makes practical use possible without additional code?